src/mudlet.h

```diff
@@ -66,6 +66,27 @@
 #include <hunspell/hunspell.hxx>
 #include <hunspell/hunspell.h>
 
+// for system physical memory info
+#ifdef WIN32
+#include <Windows.h>
+
```

Comment:
Does this empty line have any significance?

Reply:
No, removed.